php/installer/Installer.class.inc

```diff
@@ -26,9 +26,16 @@ namespace LORIS\Installer;
  */
 class Installer
 {
-
     private $_lastErr;
 
+    private const SCHEMA_FILES = ['00-schema.sql',
```

Comment:
Off-topic: I'm surprised the first array element syntax format passes phan. I would think `00-schema.sql` would need to be on a separate line.

Reply:
It is a variable declaration, not command, so I think not strict. I don't like the format like
[
'00-schema.sql
]